src/pages/DMSPolicyInfo.css

```diff
@@ -126,10 +126,6 @@
   margin-bottom: auto !important;
 }
 
-p {
-  margin-top: 10px;
-}
```

Comment:
This global style update interfered with other `p` tags across the site.

Reply:
Gah sorry about that. Rather than deleted, is it possible to scope it?  I think this might mess up the DMS page

Reply:
I think you should be able to just do:
```
.base-style p {
  margin-top: 10px;
}
```

(probably need a more dms-page specific base class)

I also noticed the same thing right below for the `b` tag

Reply:
OK, I think I have a fix for it - just have to override what's there.

Reply:
I ended up reverting this change so we can address both global styles separately

Reply:
Ah gotcha